Information about Dedicated Generation 3 (#1535)
platformsh/platformsh-docs: docs/src/configuration/services/kafka.md

|----------------------------------|---------------|
|  {{< image-versions image="kafka" status="supported" environment="grid" >}} | {{< image-versions image="kafka" status="supported" environment="dedicated" >}} |

{{< note >}}

> Apache Kafka is not available in the dedication plan.

Reviewer: ```suggestion
Apache Kafka is not available on Dedicated Production Clusters.
```

> {{< /note >}}

Reviewer: ```suggestion
{{< /note >}}

```
